src/Features/Auth/ForgetPassword/index.js

```diff
@@ -18,8 +18,7 @@ function ForgetPassword() {
     return () => {
       dispatch(setFormMessage({ message: '', status: 0 }));
     };
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
+  }, [dispatch]);
```

Comment:
I think every time component re renders dispatch will have a new reference and useEffect as compare references for object will run again please confirm that using dispatch as dependency works fine or just disable the lint here and dont use dispatch in dependency

Reply:
not running on every render, runs on mount and unmount only. I think since in dependency, it looks if the specified data changes and then runs it. But here, dispatch itself does not change, since it is defined above return of component.

Reply:
No bro dispatch is function although it is above return but the whole function body runs whenever data changes .. You can experience it by doing consoles